sql/npc_list.sql

```diff
@@ -21453,11 +21453,11 @@ INSERT INTO `npc_list` VALUES (17547501,'Achieve_Master','Achieve Master',0,0.00
 INSERT INTO `npc_list` VALUES (17547502,'Unity_Master','Unity Master',0,0.000,0.000,0.000,0,50,50,0,0,96,2,2051,0x0000340000000000000000000000000000000000,0,'SOA',0);
 INSERT INTO `npc_list` VALUES (17547507,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,2,2051,0x0000320000000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17547508,'Somnial_Threshold','Somnial Threshold',0,55.000,10.000,-74.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,NULL,1);
-INSERT INTO `npc_list` VALUES (17547509,'qm0','???',0,0.000,0.000,-101.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17547510,'qm1','???',0,0.000,-5.000,68.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17547511,'qm2','???',0,-23.000,2.000,64.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17547512,'qm3','???',0,23.000,2.000,64.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17547513,'qm4','???',0,0.000,2.000,127.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17547509,'qm0','???',0,0.000,0.000,-101.000,1,50,50,0,0,0,2,3,0x000034003F3F3F00000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17547510,'qm1','???',0,0.000,-5.000,68.000,1,50,50,0,0,0,2,3,0x000034003F3F3F00000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17547511,'qm2','???',0,-23.000,2.000,64.000,1,50,50,0,0,0,2,3,0x000034003F3F3F00000000000000000000000000,0,NULL,0);
```

Comment:
this is infilling unused bytes here. Retail doesn't zero them out so you get value sin the mask that don't mean anything. that 0x00003400 before it is the actual look and the 3f3f3f does nothing here.

Reply:
if I remove the 3f3f3f, the ??? do not appear and i can't target it.

Reply:
Ok, i change it and i set status to 0 otherwise it does not appear

Reply:
status controls the state of the object, including spawned or despawned. that's the field making it not appear as a target. the model is the same in both cases here though.

Reply:
https://github.com/topaz-next/topaz/blob/release/src/map/entities/baseentity.h#L48

Reply:
Ok thank you